### ISSUE TITLE
URL builder expects parameters even for simple routes

### ISSUE THREAD

**Issue by slavafomin** [alubbe/named-routes#17]
Hello!

Thank you for this great library!

However, for some reason `app.namedRoutes.build` function expects second argument even for routes without parameters.

Right now it throws the following exception:

```txt
/node_modules/named-routes/router.js:111
    var keys = Object.keys(userParams);
                      ^
TypeError: Cannot convert undefined or null to object
    at Function.keys (native)
    at Route.generate (/node_modules/named-routes/router.js:111:23)
    at Router.build (/node_modules/named-routes/router.js:135:33)
    ...
```

I have to pass an empty object as a workaround, i.e. `{}`.

Thanks!

**Comment by alubbe** [comment]
Fixed with https://github.com/alubbe/named-routes/commit/ac5a51010fd9afbed0c8cbdae94c5de8cedd0617, released as 2.0.5.

**Comment by slavafomin** [comment]
That was fast, thanks! = )
